packages/application-components/src/components/modal-pages/internals/modal-page.styles.ts

```diff
@@ -5,14 +5,15 @@ export const TRANSITION_DURATION = 200;
 
 type StyleProps = {
   zIndex?: number;
+  size?: 'small' | 'large';
 };
 
 export const getContainerStyles = (_props: StyleProps): SerializedStyles => css`
   position: absolute;
   top: 0;
   right: 0;
   height: 100%;
-  width: 100%;
+  width: ${_props.size === 'small' ? '600px !important' : '100%'};
```

Comment:
Wondering if we should use `maxWidth` instead of a fixed width? What happens when you resize your viewport?

Reply:
I think we do want to use a fixed size. If we use `max-width`, the panel can be narrower than we want it to be.

Anyway, it can be addressed as a follow-up when Natalia is back so I can double check with her.

[App-kit-playground - e2e-testaccount-39 - Merchant Center.webm](https://github.com/commercetools/merchant-center-application-kit/assets/97907068/0a91b3f3-23bb-40a4-b7aa-158e5e2b4cb1)

Reply:
Ok, no problem. Thanks for double checking.

I think it's fine then, 600px is small enough to be within the "minimal supported" viewport.